README.md

```diff
@@ -46,6 +46,26 @@ Please note that `era-test-node` is still in its **alpha** stage. Some features
 
 ## 📥 Installation & Setup
 
+### Using the installation script
```

Comment:
Perhaps to be added in another PR, but the recommended way for installations like this is `npx zksync-cli dev start`

Reply:
Looks like the docs are not in-sync with this README anymore: https://docs.zksync.io/build/test-and-debug/in-memory-node